darktrace/help.md

```diff
@@ -104,7 +104,7 @@ Example input:
 {
   "did": 1,
   "frequency": 300,
-  "minscore": 0,
+  "minscore": 0.1,
```

Comment:
@r7-kszczepanskagorna Let's change this back to `integer` and do the math in the code so that a user does not have to enter fractions. According to the DarkTrace docs, 0.x is x%. e.g. 0.6 is 60%. User should just have to enter a percentage e.g. 50